Extension/src/common.ts

```diff
@@ -534,6 +534,33 @@ export function checkInstallLockFile(): Promise<boolean> {
     return checkFileExists(getInstallLockPath());
 }
 
+/** Get the platform that the installed binaries belong to.*/
+export function getInstalledBinaryPlatform(): Promise<string> {
+    return new Promise(async (resolve, reject) => {
```

Comment:
Let's avoid creating explicit Promises.  If you mark this function as `async` you can use `await` directly and there's no need to call `resolve` or `reject`.  Just return the result if it succeeds and throw a `new Error` if it fails.